### DISCUSSION TITLE
Check uniqueness of related value before the parent item is saved

### DISCUSSION THREAD

**Discussion by yamiteru** [directus/directus#5197] [Feature Requests]
Right now it's possible to click "Add new" in O:M, fill an already existing value and add it and it doesn't immediately throw an error saying it already exists (if the value is unique). It throws the error when I want to save the whole thing.

It would need to asynchronously check that the new related value is unique before the parent item is saved (eg: on modal close/save).

**Comment by br41nslug** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
